__tests__/app.test.js

```diff
@@ -211,3 +211,53 @@ describe("GET /api/articles", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("status:200 responds with an array of comments for the requested article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+        .then(({body: comments}) => {
+        expect(comments).toBeInstanceOf(Array);
+      });
+  });
+  test("status:200 array of comments should have correct object keys", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: comments }) => {
+        comments.forEach((comment) => {
```

Comment:
If you're going to use a forEach you should be asserting that the array has a length of at least one.